src/wayland/wl.c

```diff
@@ -70,17 +71,20 @@ struct wl_ctx {
         } pointer;
 
 	struct {
-		struct wl_touch *wl_touch;
-		struct {
-			int32_t x, y;
-		} pts[MAX_TOUCHPOINTS];
-	} touch;
+                struct wl_touch *wl_touch;
```

Comment:
Please change the tabs to spaces. In the rest of the code as well

Reply:
Done (I think - tried to do it automatically w/ nvim and it looks ok, but probably worth double checking)